### PR TITLE
Shop links section

### PR DESCRIPTION
**Pull request by Geljinskiy**

### CODE REVIEW

File: src/components/sections/ShopLinks/ShopLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import css from './ShopLinks.module.scss';
+import { links } from 'data';
+
+const ShopLinks: React.FC = (): JSX.Element => {
+  return (
+    <Container>
+      <Section className={css.section}>
+        <ul className={css.list}>
+          {links.map(link => {
+            const { name, images, target } = link;
```
Comment (myroslavsuprun) [inline src/components/sections/ShopLinks/ShopLinks.tsx:15]: You can just pass the whole object into the component props, which is more efficient. 

But it is OK as well.

File: src/components/sections/ShopLinks/ShopLink/ShopLink.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+//   // desktop: string;
+// };
+
+// type ShopLinkProps = {
+//   name: string;
+//   images: any;
+//   target: string;
+// };
+
+type Device = {
```
Comment (myroslavsuprun) [inline src/components/sections/ShopLinks/ShopLink/ShopLink.tsx:20]: extract types into separate structure 
/types
 - index.ts
 - common.ts
 - shopLinks.ts 
 
 Or somewhere locally to the component into a separate file.
 
 imho
 1 interface inside a react component file is good, but two or more is a little overwhelming.

**Review by myroslavsuprun — APPROVED** [review]

File: src/components/sections/ShopLinks/ShopLinks.tsx (hunk 1 above)
Comment (Geljinskiy) [reply src/components/sections/ShopLinks/ShopLinks.tsx:15]: fixed on another branch - https://github.com/myroslavsuprun/audiophile/tree/feature/form-elements

File: src/components/sections/ShopLinks/ShopLink/ShopLink.tsx (hunk 2 above)
Comment (Geljinskiy) [reply src/components/sections/ShopLinks/ShopLink/ShopLink.tsx:20]: fixed on another branch - https://github.com/myroslavsuprun/audiophile/tree/feature/form-elements

**Pull request merged**